override_extension_draw.py

```diff
@@ -0,0 +1,522 @@
+            col_b.label(
+                text=exui.size_as_fmt_string(item_remote.archive_size), translate=False
+            )
+
+        col_a.label(text="License")
+        col_b.label(text=item.license, translate=False)
+
+        col_a.label(text="Repository")
+        col_b.label(text=repo_item.name, translate=False)
+
+        if is_installed:
+            col_a.label(text="Path")
+            col_b.label(text=os.path.join(repo_item.directory, pkg_id), translate=False)
+
+
+def extension_draw_item_override(
+    layout,
+    *,
+    pkg_id,  # `str`
+    item_local,  # `PkgManifest_Normalized | None`
+    item_remote,  # `PkgManifest_Normalized | None`
+    is_enabled,  # `bool`
+    is_outdated,  # `bool`
+    show,  # `bool`.
+    mark,  # `bool | None`.
+    # General vars.
+    repo_index,  # `int`
+    repo_item,  # `RepoItem`
+    operation_in_progress,  # `bool`
+    extensions_warnings,  # `dict[str, list[str]]`
+):
+    # filter by verification state, only for blenderkit repository
+    if repo_item.remote_url == EXTENSIONS_API_URL:
+        # ensure cache is loaded
+        ensure_repo_cache()
+        # get the cache
+        bk_ext_cache = bpy.context.window_manager["blenderkit_extensions_repo_cache"]
+        bk_cache_pkg = bk_ext_cache.get(pkg_id[:32], None)
+        # get the filter setting
+        search_verification_status = (
+            bpy.context.window_manager.blenderkit_extension_validation_settings.search_verification_status
+        )
+        # filter blenderkit packages based on verification status
+
+        if search_verification_status != "ALL":
+            if (
+                bk_cache_pkg is not None
+                and not bk_cache_pkg.get("verification_status")
+                == search_verification_status.lower()
+            ):
+                layout.scale_y = 0.01
+                layout.enabled = False
+                layout.emboss = "NONE"
+
+                return False
+        extension_draw_item_blenderkit(
+            layout,
+            pkg_id=pkg_id,
+            item_local=item_local,
+            item_remote=item_remote,
+            is_enabled=is_enabled,
+            is_outdated=is_outdated,
+            show=show,
+            mark=mark,
+            repo_index=repo_index,
+            repo_item=repo_item,
+            operation_in_progress=operation_in_progress,
+            extensions_warnings=extensions_warnings,
+        )
+        return True
+
+    exui.extension_draw_item_original(
+        layout,
+        pkg_id=pkg_id,
+        item_local=item_local,
+        item_remote=item_remote,
+        is_enabled=is_enabled,
+        is_outdated=is_outdated,
+        show=show,
+        mark=mark,
+        repo_index=repo_index,
+        repo_item=repo_item,
+        operation_in_progress=operation_in_progress,
+        extensions_warnings=extensions_warnings,
+    )
+
+    return True
+
+
+def override_draw_function():
+    print("overriding extension draw function")
```

Comment:
Are we sure we want to print this?

Reply:
I'm sure itwas quite stupid to leave it there ;)